install_cli_tools.sh

```diff
@@ -0,0 +1,12 @@
+#!\bin\sh
+curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+unzip awscli-bundle.zip
+./awscli-bundle/install -b ~/bin/aws
+export PATH=~/bin:$PATH
+export tf_version=0.12.20 tf_init_cli_options="-input=false" tf_validation_cli_options="" tf_plan_cli_options="-lock=false -input=false" tf_apply_cli_options="-auto-approve -input=false"
```

Comment:
I would opt to store this somewhere in the current working directory instead of in the home directory since this is being run in CI.